javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -569,23 +563,13 @@ private Vector(BitMappedTrie<T> trie) {
     }
 
     @Override
-    public Vector<T> append(T element) {
-        return wrap(trie.append(element));
-    }
+    public Vector<T> append(T element) { return appendAll(Iterator.of(element)); }
```

Comment:
a single `append` is almost `3x` slower than before, but the bulk operations (`*all`) are `~17x` faster than before, making it faster than `Scala`